.travis/package.sh

```diff
@@ -14,8 +14,13 @@ if [ "$TRAVIS_OS_NAME" = "linux" ] && [ "$PACKAGE" = "ON" ]; then
     mv ~/out/* .
 fi
 
+declare -A urls
+
 for file in SuperTux*; do
     echo "Uploading $file";
-    curl --upload-file "$file" "https://transfer.sh/$file"
-    echo
+    url=$(curl --upload-file "$file" "https://transfer.sh/$file")
```

Comment:
Please add notes to the concerned files on the page that they'll only be available for 2 weeks. Since Travis CI builds every week (using cron), there should always be a new one available though.